app/src/main/java/org/schabi/newpipe/fragments/list/search/SearchFragment.java

```diff
@@ -386,7 +387,14 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         int itemId = 0;
         boolean isFirstItem = true;
         final Context c = getContext();
-        for(String filter : service.getSearchQHFactory().getAvailableContentFilter()) {
+        for (String filter : service.getSearchQHFactory().getAvailableContentFilter()) {
+            if (filter.equals("music_songs")) {
+                MenuItem musicItem = menu.add(2,
+                        itemId++,
+                        0,
+                        "YouTube Music");
+                musicItem.setEnabled(false);
+            }
```

Comment:
Yeah, adding a disabled item as a header title is bad...

Though I don't know if something like that can even be implemented in a native way (closest thing would be a sub-menu), and I'm not sure if it's worth it to add custom components when this option will be improved in the near future.

Will we just leave it like this until that day?

Reply:
No, this couldn't be implemented in a native way other than submenus. I told Tobias that I didn't really like using a disabled item for that, but he seems to be fine with it until we improve the whole search filter thing.

Reply:
> Yeah, adding a disabled item as a header title is bad...

Yes. I hope that we will get proper search filter soon.